src/Resources/views/index.blade.php

```diff
@@ -9,7 +9,7 @@
                     @foreach($columns as $column)
                         <th>
                             <a href="#sort">
-                                {!! ucfirst($column) !!}
+                                {!! title_case(str_replace("_", " ", snake_case($column))) !!}
```

Comment:
What if instead of function chain introduce some directive? It will be useful also in src/Resources/views/show.blade.php:8. What do you think?